documentation/content/en/books/handbook/wine/_index.adoc

```diff
@@ -556,7 +556,7 @@ image::winetricks-uninstall-3.png[]
 [[homura]]
 === Mizutamari
 
-Mizutamari is an application similar to `winetricks`, although it was inspired by the https://lutris.net/[Lutris] gaming system for Linux.
+https://codeberg.org/Alexander88207/Mizutamari[Mizutamari] is an application similar to `winetricks`, although it was inspired by the https://lutris.net/[Lutris] gaming system for Linux.
```

Comment:
It's also available in the ports collection as games/mizuma, but the link to its homepage seems more appropriate than the ports collection entry, for this purpose, so I left it alone.